### ISSUE TITLE
Impossible to insert text after table at page end

### ISSUE THREAD

**Issue by maddin18** [cosmocode/dokuwiki-plugin-prosemirror#123]
In case I insert a table at the end of a page, I can't place the cursor behind this table. Therefore it is impossible to insert text behind/below the table in WYSIWYG mode.

**Comment by vic-t** [comment]
This is quite a nuisance, indeed. And it does clearly look like a bug. Because when you toggle back to DW Edit from WYSIWIG, you see that there indeed are empty lines below the table. In my case, there are 2, and even if I delete them, toggling brings them back. However, in WYSIWIG mode, I can't write below the table. That is, unless, when in DW Edit mode, I write anything below the table. 

This issue is clearly linked with https://github.com/cosmocode/dokuwiki-plugin-prosemirror/issues/125. Basically, a new line is and should always be provided below a new table that you can get to by simply pressing the down arrow key when you're in the last table row.

**Comment by Monklite** [comment]
Agreed, this has been rather annoying...

**Comment by mschlenstedt** [comment]
Yes, same here. Also below plugin blocks no editing is possible,

**Comment by Klap-in** [comment]
Because everybody has not more than 24 hours a day ;-) You are welcome to open a pull request to improve this situation. Alternative is to hire someone who helps you.

**Comment by grazianobolla** [comment]
You are right.
